JS/API/ScoreSearcher.proto.js

```diff
@@ -62,10 +62,35 @@ const factoryScoreSearcher = (musicObj) =>
   //"private" variables..note state is safest kept constant-------------------
   const scoreSearcher = {};
   const instrumentObjects = makeInstrumentObjects(musicObj);
+  let currKey = {"C": 0, "D": 0, "E": 0, "F": 0, "G": 0, "A": 0, "B": 0}; //Sharps/flats for a given note
 
   //"private" functions in scope------------------------------
-
   //...ex: function poop() { ... }
+  function setKey(fifths)
```

Comment:
This function can be made local to getAccidentals -in other words it should become an inner function when moved there